modules/ROOT/pages/security/ssl-framework.adoc

```diff
@@ -956,6 +956,29 @@ If the certificate is a different path outside of NEO4J_HOME, then set the absol
 dbms.ssl.policy.backup.client_auth=REQUIRE
 ----
 
+=== Configure the backup client for SSL
+
+When using `neo4j-admin backup` command, you need to specify the SSL policy to be used by the backup client.
```

Comment:
I think this doesn't make it entirely clear that you can back up against the cluster endpoint(6000) and its SSL policy, or the backup endpoint(6362 default) and its SSL policy, but regardless of which you do back up against, the admin client will use the SSL policy specified for backups `dbms.ssl.policy.backup.*`